### PR TITLE
Delete CSS FAQ

### PR DESCRIPTION
**Pull request by wbamberg**
Deleting https://developer.mozilla.org/en-US/docs/Web/CSS/FAQ .

I did not think this was a useful page.

### CODE REVIEW

File: files/en-us/_redirects.txt [hunk 1]
```diff
@@ -492,7 +492,7 @@
 /en-US/docs/CSS/Currentcolor	/en-US/docs/Web/CSS/color_value#currentcolor_keyword
 /en-US/docs/CSS/Descendant_selectors	/en-US/docs/Web/CSS/Descendant_combinator
 /en-US/docs/CSS/Draft_Implementations_of_CSS_Features	/en-US/docs/Web/CSS/Mozilla_Extensions
-/en-US/docs/CSS/FAQ	/en-US/docs/Web/CSS/FAQ
+/en-US/docs/CSS/FAQ	/en-US/docs//en-US/docs/Web/CSS
```
Comment (Josh-Cena) [inline files/en-us/_redirects.txt:495]: Hah! Tricks someone every time.

Comment (wbamberg) [reply files/en-us/_redirects.txt:495]: 😭

Comment (wbamberg) [inline files/en-us/_redirects.txt:495]:
```suggestion
/en-US/docs/CSS/FAQ	/en-US/docs/Web/CSS
```

File: files/en-us/_redirects.txt [hunk 2]
```diff
@@ -11212,6 +11212,7 @@
 /en-US/docs/Web/CSS/Currentcolor	/en-US/docs/Web/CSS/color_value#currentcolor_keyword
 /en-US/docs/Web/CSS/Descendant_selectors	/en-US/docs/Web/CSS/Descendant_combinator
 /en-US/docs/Web/CSS/Draft_Implementations_of_CSS_Features	/en-US/docs/Web/CSS/Mozilla_Extensions
+/en-US/docs/Web/CSS/FAQ	/en-US/docs//en-US/docs/Web/CSS
```
Comment (wbamberg) [inline files/en-us/_redirects.txt:11215]:
```suggestion
/en-US/docs/Web/CSS/FAQ	/en-US/docs/Web/CSS
```

**Review by Elchi3 — APPROVED** [review]

**Pull request merged**